src/main/java/nextstep/sessions/service/SessionService.java

```diff
@@ -28,6 +35,8 @@ public void registerSession(Long sessionId, NsUser nsUser) {
                 .orElseThrow(() -> new IllegalArgumentException("결제 정보가 없습니다."));
 
         session.register(nsUser, payment);
+        sessionRepository.save(session);
+        studentRepository.save(nsUser, session);
```

Comment:
수강신청한 결과로 Student 객체를 생성한다면 student 객체를 바로 저장하면 어떨까요?
```suggestion
        Student student = session.register(nsUser, payment);
        studentRepository.save(student);
```